### ISSUE TITLE
configure error

### ISSUE THREAD

**Issue by kengcher** [beagle-dev/beagle-lib#104]
hi 

i am trying to install the beagle-lib on my mac w gcc 4.8.5 and ran into this error:

checking cpuid.h usability... yes
checking cpuid.h presence... yes
checking for cpuid.h... yes
checking for gcj... gcj -C
checking if gcj -C works... configure: error: The Java compiler gcj -C failed (see config.log, check the CLASSPATH?)

**Comment by wsuplantpathology** [comment]
Hi @kengcher:

Did you find a solution? I was trying to install beagle-lib on my Linux, and got exactly same error as yours.

**Comment by casbie** [comment]
I have the same problem when installing beagle-lib on the server. But I successfully install on my VM, where gcj is not installed. So I just rename the executable file gcj as another name (eg. gcj-tmp), then the configure ignore the gcj check.

**Comment by MatthewMah** [comment]
As I understand it, this is a problem relating to autotools looking for [gcj](https://gcc.gnu.org/wiki/GCJ), the now unsupported GNU java compiler. If you have another java compiler available, this one should be usable instead. 

1. Modify `configure.ac` to tell autotools which java compiler to use as follows:

```
diff --git a/configure.ac b/configure.ac
index 5b92d96..648677e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -332,6 +332,7 @@ jnilibext=no
 
 if test "x$with_jdk" != "xno"
 then
+    JAVAC=javac
     AC_PROG_JAVAC
     if test "x$JAVAC" != "x"
     then  
```
In this example, I am using the `javac` compiler in the OpenJDK, and I insert a single line `JAVAC=javac`. 

2. Continue with `autogen.sh` and `configure` steps as in the instructions.

**Comment by msuchard** [comment]
Merged in e66bf15a47fa4f9dfea0cdf66e66594fb805f77d .  Let me know if this solves all issues above.  Thanks to @MatthewMah.
